tests/functional/test_article.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+
+from pages.article import ArticlePage
+
+
+# page title
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_is_title_expected(base_url, selenium):
+    page = ArticlePage(selenium, base_url).open()
+    assert page.is_title_expected()
```

Comment:
A failure here will show expected True but was False, which is less helpful than an assert that compares the actual values, which will then show the expected and actual values in the trace.